### ISSUE TITLE
[1.20.1] Chaos guardian Laser

### ISSUE THREAD

**Issue by yaar4e** [Draconic-Inc/Draconic-Evolution#1813]
## Base information
* Minecraft version: 1.20.1
* Mod version: 3.1.0-test-2.9999
* Minecraft Forge version:  47.2.1
* Mod Pack: (if applicable)

### Description / steps to reproduce

The Guardian of Chaos strikes with a laser beam, and there is no way to evade it, even behind a pillar. In addition, the Draconic Undying Module does not save.

https://youtu.be/LWJLIdsLTaM

**Comment by FoxMcloud5655** [comment]
Yes, you can block it with the pillar.  That's how it's intended to be blocked.  It also has an explosion radius, so you cannot be too close to the impact point.  The beam also kills through invulnerability, specifically to defeat stuff like the MekaSuit.  It's very easy to avoid if you take these steps.

**Comment by yaar4e** [comment]
Please watch this video starting at 16:50, is the explosion radius that big? https://www.youtube.com/watch?v=Alq4y6wACMk

**Comment by FoxMcloud5655** [comment]
No, it's not that big.  The beam isn't even firing, meaning either you were lagging pretty hard and the server "caught up" all at once and killed you, your simulation distance is set so low that the Chaos Guardian wasn't loaded, or there's something else going on.

Considering you have a custom build of Draconic Evolution and not an official build from CurseForge, we cannot verify that this issue is actually an issue.  If you're editing the mod to include Russian translations, then just add your translation to a resource pack.  There's no need to recompile the entire mod.
